app/konnect-audit-logs.md

```diff
@@ -0,0 +1,332 @@
+---
+title: "{{site.konnect_short_name}} audit logs"
+content_type: reference
+layout: reference
+
+products:
+    - gateway
+
+description: Use {{site.konnect_short_name}} audit logs to help you detect and respond to potential security incidents when they occur.
+
+related_resources:
+  - text: "Set up an audit log webhook for a {{site.konnect_short_name}} org"
+    url: /how-to/set-up-konnect-org-audit-log-webhook/
+  - text: "Set up an audit log webhook for Dev Portal"
+    url: /how-to/set-up-dev-portal-audit-log-webhook/
+  - text: "{{site.konnect_short_name}} Org audit log API"
+    url: /api/konnect/audit-logs/v2/
+  - text: "Dev Portal audit log API"
+    url: /api/konnect/portal-management/v2/#/operations/update-portal-audit-log-replay-job
+  - text: "{{site.base_gateway}} audit logs"
+    url: /gateway/audit-logs/
+  - text: "{{site.base_gateway}} logs"
+    url: /gateway/logs/
+---
+
+Audit logs can help you detect and respond to potential security incidents when they occur. Monitoring audit logs proactively can reduce the risk of outages and ensure continuous service for your users. No system can ever be completely secure, but audit logs can be a key part of your incident prevention infrastructure.
+
+Audit logging provides the following benefits:
+* **Security**: System events can be used to show abnormalities to be investigated, forensic information related to breaches, or provide evidence for compliance and regulatory purposes.
+* **Compliance**: Regulators and auditors may require audit logs to confirm whether certain certification standards are met.
+* **Debugging**: Audit logs can help determine the root causes of efficiency or performance issues.
+* **Risk management**: Prevent issues or catch them early.
+
+{:.warning}
+> Org-wide {{site.konnect_short_name}} audit logs are set up and managed separately from Dev Portal audit logs.
```

Comment:
```suggestion
```

Until we know what Dev Portal looks like after the big transition they are working on I think we can avoid drawing attention to it.